### ISSUE TITLE
Docs wont generate with Skript 2.6+

### ISSUE THREAD

**Issue by DampfWaffel** [Skylyxx/SkriptDocsGenerator#4]
Hello,

due to some changes this Plugins dont generate Docs anymore.
```
[19:34:53] [Server thread/WARN]: Unexpected exception while parsing console command "docs"
org.bukkit.command.CommandException: Unhandled exception executing command 'docs' in plugin SkriptDocsGenerator v1.2
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[Skigot.1.0.jar:git-Spigot-9fb885e-296df56]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:145) ~[Skigot.1.0.jar:git-Spigot-9fb885e-296df56]
        at org.bukkit.craftbukkit.v1_16_R3.CraftServer.dispatchCommand(CraftServer.java:732) ~[Skigot.1.0.jar:git-Spigot-9fb885e-296df56]
        at org.bukkit.craftbukkit.v1_16_R3.CraftServer.dispatchServerCommand(CraftServer.java:717) ~[Skigot.1.0.jar:git-Spigot-9fb885e-296df56]
        at net.minecraft.server.v1_16_R3.DedicatedServer.handleCommandQueue(DedicatedServer.java:411) ~[Skigot.1.0.jar:git-Spigot-9fb885e-296df56]
        at net.minecraft.server.v1_16_R3.DedicatedServer.b(DedicatedServer.java:380) ~[Skigot.1.0.jar:git-Spigot-9fb885e-296df56]
        at net.minecraft.server.v1_16_R3.MinecraftServer.a(MinecraftServer.java:1071) ~[Skigot.1.0.jar:git-Spigot-9fb885e-296df56]
        at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:910) ~[Skigot.1.0.jar:git-Spigot-9fb885e-296df56]
        at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$0(MinecraftServer.java:256) ~[Skigot.1.0.jar:git-Spigot-9fb885e-296df56]
        at java.lang.Thread.run(Thread.java:831) [?:?]
Caused by: java.lang.NoSuchMethodError: 'java.lang.String ch.njol.skript.classes.ClassInfo.getDocumentationId()'
        at fr.skylyxx.docsgenerator.utils.DocBuilder.generateClassInfoDoc(DocBuilder.java:103) ~[?:?]
        at fr.skylyxx.docsgenerator.utils.DocBuilder.generateAddonDoc(DocBuilder.java:182) ~[?:?]
        at fr.skylyxx.docsgenerator.commands.SkriptDocsGeneratorCommand.onCommand(SkriptDocsGeneratorCommand.java:33) ~[?:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[Skigot.1.0.jar:git-Spigot-9fb885e-296df56]
        ... 9 more
```
       
The Name of the methode got changed https://github.com/SkriptLang/Skript/commit/7a1929aa22a301e0a1386ace92429b114f0bc777 from "getDocumentationI**d**" to "getDocumentationI**D**"

Skript: 2.6-beta3

Grats

**Comment by Skylyxx** [comment]
Hi, first, thanks for your report.
I'm currently in holydays. I will fix that as soon as possible. 
Skylyxx

**Comment by Skylyxx** [comment]
Fixed in Skylyxx/SkriptDocsGenerator@6fc0dd8
